docs/source/spatialstats.rst

```diff
@@ -6,16 +6,15 @@ Spatial statistics
 Spatial statistics, also referred to as `geostatistics <https://en.wikipedia.org/wiki/Geostatistics>`_, are essential
 for the analysis of observations distributed in space.
 To analyze DEMs, xDEM integrates spatial statistics tools specific to DEMs described in recent literature,
-in particular in `Rolstad et al. (2009) <https://doi.org/10.3189/002214309789470950>`_,
-`Dehecq et al. (2020) <https://doi.org/10.3389/feart.2020.566802>`_ and
-`Hugonnet et al. (2021) <https://doi.org/10.1038/s41586-021-03436-z>`_. The implementation of these methods relies
-partly on the package `scikit-gstat <https://mmaelicke.github.io/scikit-gstat/index.html>`_.
+in particular in `Hugonnet et al. (2022) <https://doi.org/10.1109/jstars.2022.3188922>`_ and
+`Rolstad et al. (2009) <https://doi.org/10.3189/002214309789470950>`_. The implementation of these methods relies
+partially on the package `scikit-gstat <https://mmaelicke.github.io/scikit-gstat/index.html>`_.
 
 The spatial statistics tools can be used to assess the precision of DEMs (see the definition of precision in :ref:`intro`).
 In particular, these tools help to:
 
-    - account for non-stationarities of elevation measurement errors (e.g., varying precision of DEMs with terrain slope),
-    - quantify the spatial correlation of measurement errors in DEMs (e.g., native spatial resolution, instrument noise),
+    - account for elevation heteroscedasticity (e.g., varying precision with terrain slope),
+    - quantify the spatial correlation of errors in DEMs (e.g., native spatial resolution, instrument noise),
     - estimate robust errors for observations integrated in space (e.g., average or sum of samples),
```

Comment:
add, "to convert elevation change to volume change" to make it more easy to understand?

Reply:
Done